src/app/core/public/api/impl/Circuit.ts

```diff
@@ -195,14 +195,27 @@ export abstract class CircuitImpl implements Circuit {
     // Wire connection can fail if i.e. p1 is reference-equal to p2
     public abstract connectWire(p1: Port, p2: Port): Wire | undefined;
 
+    /** Will delete all components and wires
+     * contained within the Obj[] array
+     * Ignores ports as deleting ports is a no-op
```

Comment:
Let's be more specific in that it won't delete ports directly, but will delete ports that are apart of a component that's in `objs`